### PR TITLE
imagemagick: fixed CVE-2023-5341

### PR DESCRIPTION
**Pull request by rawlingsj**
patch released upstream https://github.com/ImageMagick/ImageMagick/commit/aa673b2e4defc7cad5bec16c4fc8324f71e531f1

### CODE REVIEW

**Review by cpanato — CHANGES_REQUESTED** [review]
can you rebase once https://github.com/wolfi-dev/advisories/pull/743 is merged?

**Comment by cpanato** [comment]
ping :)

**Comment by rawlingsj** [comment]
All done

File: imagemagick.advisories.yaml [hunk 1]
```diff
@@ -698,6 +698,10 @@ advisories:
     aliases:
       - GHSA-gp6r-24h3-qcjv
     events:
+      - timestamp: 2024-01-07T18:27:11Z
+        type: fixed
+        data:
+          fixed-version: 7.1.1.25-r0
```
Comment (cpanato) [inline imagemagick.advisories.yaml:704]: should be in the end

Comment (cpanato) [reply imagemagick.advisories.yaml:704]: i will rebase

**Review by cpanato — APPROVED** [review]

**Pull request merged**